examples/image-vis-type-example.ipynb

```diff
@@ -14,7 +14,8 @@
     "\n",
     "```\n",
     "pip install mapboxgl\n",
-    "pip install Pillow numpy\n",
+    "pip install imread\n",
```

Comment:
@ryanbaumann `imread` comes from `matplotlib` which is already installed https://github.com/mapbox/mapboxgl-jupyter/blob/cf08d3e2fe69e866838cb495d83380d436c4f0dc/setup.py#L34 so no need to install `imread`